variables.tf

```diff
@@ -1,4 +1,11 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+variable "iam_source_json_url" {
+  type        = string
+  description = "IAM source json policy to download"
```

Comment:
```suggestion
  description = "IAM source JSON policy to download"
```

Reply:
explain (in the description and prob in README, why do we need the URL to download external policy JSON files.
Mention that it's used in `source_json`